### PR TITLE
Bootstrap split

### PR DESCRIPTION
**Pull request by teknex**

### CODE REVIEW

File: src/android/java/org/alias/wallet/BootstrapService.java [hunk 1]
```diff
@@ -43,11 +49,24 @@
...
     public static final int STATE_FINISHED = 3;
+    public static final int STATE_CANCEL = -1;
+    public static final int STATE_ERROR = -2;
+
+    public static final int ERROR_NOSPACE_DOWNLOAD = 1;
+    public static final int ERROR_NOSPACE_EXTRACTION = 2;
+    public static final int ERROR_EXTRACTION = 3;
+    public static final int ERROR_HASH = 4;
+    public static final int ERROR_BOOTSTRAP_INDEX_404 = 5;
+    public static final int ERROR_BOOTSTRAP_FILE_404 = 6;
```
Comment (HLXEasy) [inline src/android/java/org/alias/wallet/BootstrapService.java]: Why not an enum?

File: src/android/java/org/alias/wallet/BootstrapService.java [hunk 2]
```diff
@@ -7,33 +7,41 @@
...
+import java.time.Clock;
+import java.time.Duration;
+import java.util.ArrayList;
 import java.util.Comparator;
-import java.util.Enumeration;
-import java.util.zip.ZipEntry;
-import java.util.zip.ZipFile;
+import java.util.List;
+
+import static org.alias.wallet.BootstrapService.BootstrapError.*;
```
Comment (HLXEasy) [inline src/android/java/org/alias/wallet/BootstrapService.java:44]: Why this import? The enum is defined inside of the class!?

Comment (teknex) [reply src/android/java/org/alias/wallet/BootstrapService.java:44]: Without the static import, you have to prefix the enum constants with the enum type when using them

Comment (HLXEasy) [reply src/android/java/org/alias/wallet/BootstrapService.java:44]: True, ok.

**Pull request merged**